src/components/tooltip/Tooltip.tsx

```diff
@@ -325,12 +325,10 @@ class Tooltip extends React.Component<TooltipProps, State> {
                 onContextMenu={this.handleTooltipEvent}
                 onKeyPress={this.handleTooltipEvent}
             >
-                <div role="tooltip" aria-live="polite">
-                    {tooltipInner}
-                </div>
+                <div aria-live="polite">{tooltipInner}</div>
             </div>
         ) : (
-            <div className={classes} id={this.tooltipID} role="tooltip" aria-live="polite">
```

Comment:
Should we be removing from every tooltip or just error types?

Reply:
It should be removed for every tooltip, it won't affect screen reader announcements in any meaningful way `aria-describedby` and `aria-labelledby` do all the heavy lifting.

Reply:
Is this true @fsuarez-globant? I'm a bit leery about removing this `role` from all Tooltips unless we have firm direction that it's not needed. Some brief research gave me these links:

- Having `role` on the tooltip - https://inclusive-components.design/tooltips-toggletips/#tooltipasprimarylabel 
- Inconsistencies with `aria-label`, `aria-labelledby`, and `aria-describedby` - https://www.tpgi.com/short-note-on-aria-label-aria-labelledby-and-aria-describedby/